diff -Naur a/package/gettext/gettext.mk b/package/gettext/gettext.mk
--- a/package/gettext/gettext.mk	2019-08-14 00:20:05.301864046 +0200
+++ b/package/gettext/gettext.mk	2019-08-14 00:20:05.301864046 +0200
@@ -1,3 +1,5 @@
+GETTEXT_CONF_ENV = \
+        CFLAGS="$(TARGET_CFLAGS) -fPIC"
 ################################################################################
 #
 # gettext
diff -Naur a/package/gmp/gmp.mk b/package/gmp/gmp.mk
--- a/package/gmp/gmp.mk	2019-08-14 00:20:05.305864070 +0200
+++ b/package/gmp/gmp.mk	2019-08-14 00:20:05.309864095 +0200
@@ -1,3 +1,5 @@
+GMP_CONF_ENV = \
+        CFLAGS="$(TARGET_CFLAGS) -fPIC"
 ################################################################################
 #
 # gmp
diff -Naur a/package/libiconv/libiconv.mk b/package/libiconv/libiconv.mk
--- a/package/libiconv/libiconv.mk	2019-08-14 00:20:05.293863996 +0200
+++ b/package/libiconv/libiconv.mk	2019-08-14 00:20:05.293863996 +0200
@@ -1,3 +1,5 @@
+LIBICONV_CONF_ENV = \
+        CFLAGS="$(TARGET_CFLAGS) -fPIC"
 ################################################################################
 #
 # libiconv
