diff -Naur a/package/gdb/gdb.mk b/package/gdb/gdb.mk
--- a/package/gdb/gdb.mk	2019-08-14 08:37:07.898137864 +0200
+++ b/package/gdb/gdb.mk	2019-08-14 08:37:07.902137841 +0200
@@ -1,3 +1,6 @@
+TARGET_CFLAGS += -fPIC
+GDB_CONF_ENV = \
+        CFLAGS="$(TARGET_CFLAGS)"
 ################################################################################
 #
 # gdb
diff -Naur a/package/ipsec-tools/ipsec-tools.mk b/package/ipsec-tools/ipsec-tools.mk
--- a/package/ipsec-tools/ipsec-tools.mk	2019-08-14 08:37:07.946137582 +0200
+++ b/package/ipsec-tools/ipsec-tools.mk	2019-08-14 08:37:07.950137558 +0200
@@ -1,3 +1,6 @@
+TARGET_CFLAGS += -fPIC
+IPSEC_TOOLS_CONF_ENV = \
+        CFLAGS="$(TARGET_CFLAGS)"
 ################################################################################
 #
 # ipsec-tools
